tck/methods/account.ts

```diff
@@ -81,3 +86,79 @@ export const createAccount = async ({
         status: receipt.status.toString(),
     };
 };
+
+export const updateAccount = async ({
+    accountId,
+    key,
+    autoRenewPeriod,
+    expirationTime,
+    receiverSignatureRequired,
+    memo,
+    maxAutoTokenAssociations,
+    stakedAccountId,
+    stakedNodeId,
+    declineStakingReward,
+    commonTransactionParams,
+}: UpdateAccountParams): Promise<AccountResponse> => {
+    let transaction = new AccountUpdateTransaction().setGrpcDeadline(30000);

```

Comment:
This 30000 number is not explicit and it's repetitive. Maybe separate it to a const. `const DEFAULT_GRPC_DEADLINE= 30000 `?